src/content/blog/drupalcon-barcelona-2024.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "DrupalCon Barcelona Wrap-up"
+pubDate: 2024-09-30
+#modifiedDate: 2024-09-06
+summary: DDEV happenings at DrupalCon Barcelona
+author: Randy Fay
+#featureImage:
+#  src: /img/blog/2024/09/ddev-blog.png
+#  alt: DDEV running in the GitLab Pipeline
+#  credit: "[Milad Fakurian](https://unsplash.com/de/@fakurian) (Background)"
+categories:
+  - Community
+---
+
+I spent the last week in Barcelona, Spain for DrupalCon Barcelona. It was so great to meet old friends and new, and to workk with all to integrate new contributors to Drupal.
+
+## BoFs (Birds-of-a-feature) sessions
+
+A cool thing about DrupalCon is that anybody can set up a mini-session, and no real approval process is needed. I typically propose several of these, since they're a great way to deal with niche topics with a few people and have nice, smaller conversations with people who have specific needs. This time I did:
+
+* **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces). 
```

Comment:
I see that there is a new different approach in Codespaces, but we are going to make a new release soon, should we use the latest link here?

```suggestion
* **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/#ddev-installation-codespaces). 
```

Reply:
I think I prefer the latest link for right now. Does no harm for the future. Thanks for flagging.